### PR TITLE
[LC-1047] Use only valid votes when leader creates block

### PR DESCRIPTION
**Pull request by metalg0su**

### CODE REVIEW

File: loopchain/blockchain/blocks/v1_0/block_factory.py [hunk 1]
```diff
@@ -73,7 +74,10 @@ async def create_data(self, data_number: int, prev_id: bytes, epoch_num: int, ro
         block_builder.validators_hash = pre_invoke_response.validators_hash
 
         if prev_votes:
-            prev_vote = prev_votes[0]  # FIXME
+            try:
+                prev_vote = next(vote for vote in prev_votes if vote.is_real())
+            except StopIteration:
+                raise RuntimeError("Something wrong...") from StopIteration
```
Comment (metalg0su) [inline loopchain/blockchain/blocks/v1_0/block_factory.py:80]: Need useful msg and exc type.
Please comment on this.

Comment (windies21) [reply loopchain/blockchain/blocks/v1_0/block_factory.py:80]: If the prev_votes are checked then this exception cannot be reached.

Comment (windies21) [reply loopchain/blockchain/blocks/v1_0/block_factory.py:80]: So.... Just change the message.... How about this?
RuntimeError ("Unverified prev_votes.")

Comment (metalg0su) [reply loopchain/blockchain/blocks/v1_0/block_factory.py]: Applied!

**Review by windies21 — APPROVED** [review]

**Pull request merged**